README.md

```diff
@@ -1,3 +1,636 @@
+
+```js
+webhooks.removeHandler(eventName, handler)
+webhooks.removeHandler(eventNames, handler)
+```
+
+<table width="100%">
+  <tr>
+    <td>
+      <code>
+        eventName
+      </code>
+      <em>
+        String
+      </em>
+    </td>
+    <td>
+      <strong>Required.</strong>
+      Name of the event. One of <a href="#listofalleventnames">GitHub’s supported event names</a>.
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <code>
+        eventNames
+      </code>
+      <em>
+        Array
+      </em>
+    </td>
+    <td>
+      <strong>Required.</strong>
+      Array of event names.
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <code>
+        handler
+      </code>
+      <em>
+        Function
+      </em>
+    </td>
+    <td>
+      <strong>Required.</strong>
+      Method which was previously passed to <code><a href="webhookson">webhooks.on()</a></code>. If the same handler was registered multiple times for the same event, only the most recent handler gets removed.
+    </td>
+  </tr>
+</table>
+
+The `.removeHandler()` method belongs to the [receiver](receiver/) module which can be used standalone.
+
+### webhooks.middleware()
+
+```js
+webhooks.middleware(request, response[, next])
+```
+
+<table width="100%">
+  <tr>
+    <td>
+      <code>
+        request
+      </code>
+      <em>
+        Object
+      </em>
+    </td>
+    <td>
+      <strong>Required.</strong>
+      A Node.js <a href="https://nodejs.org/docs/latest/api/http.html#http_class_http_clientrequest">http.ClientRequest</a>.
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <code>
+        response
+      </code>
+      <em>
+        Object
+      </em>
+    </td>
+    <td>
+      <strong>Required.</strong>
+      A Node.js <a href="https://nodejs.org/docs/latest/api/http.html#http_class_http_serverresponse">http.ServerResponse</a>.
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <code>
+        next
+      </code>
+      <em>
+        Function
+      </em>
+    </td>
+    <td>
+      Optional function which invokes the next middleware, as used by <a href="https://github.com/senchalabs/connect">Connect</a> and <a href="http://expressjs.com/">Express</a>.
+    </td>
+  </tr>
+</table>
+
+Returns a `requestListener` (or _middleware_) method which can be directly passed to [`http.createServer()`](https://nodejs.org/docs/latest/api/http.html#http_http_createserver_requestlistener), <a href="http://expressjs.com/">Express</a> and other compatible Node.js server frameworks.
+
+Can also be used [standalone](middleware/).
+
+### List of all event names
+
+See the full list of [event types with example payloads](https://developer.github.com/v3/activity/events/types/).
+
+If there are actions for a webhook, events are emitted for both, the webhook name as well as a combination of the webhook name and the action, e.g. `installation` and `installation.created`. A `*` event is always emitted. If an event handler throws an error an `error` event is triggered.
```

Comment:
When I first started with probot listening on `*` events was a confusing concept for me, so I think maybe this wording could be expanded to explain listening on * listens to all events. Not sure if it should be included in the chart though?

Reply:
Thanks for bringing this up. I agree we should make this more clear. The difference between `*` and `error` event and all the others is that these two are implemented on the library level, all the others come from GitHub webhooks. I’ll try to make this more clear by adding a section "Special events", I’m open to suggestions :)

Reply:
👍 I think a "Special Events" category would cover it well, and allow you to explain in more detail how `*` and `error` work.